tsconfig.json

```diff
@@ -15,7 +15,7 @@
     "node_modules"
   ],
   "typedocOptions": {
-    "mode": "file",
+    "entryPointStrategy": "expand",
```

Comment:
`entryPointStrategy` = `expand` used to be the default in older versions of typedoc, so this retains the same behavior in the current version.